packages/aragon-wrapper/src/index.js

```diff
@@ -873,6 +873,31 @@ export default class Aragon {
       } catch (_) { }
     }
 
+    return this.calculateDirectPath (sender, destination, directTransaction, finalForwarder)
+
+  }
+
+  /**
+   * Calculate the transaction path for a transaction to `destination`
+   * that invokes `directTransaction`.
+   *
+   * @param  {string} sender
+   * @param  {string} destination
+   * @param  {Object} directTransaction
+   * @param  {string} [finalForwarder]
+   * @return {Array<Object>} An array of Ethereum transactions that describe each step in the path
+   */
+  async calculateDirectPath (sender, destination, directTransaction, finalForwarder) {
+    const minGasPrice = this.web3.utils.toWei('20', 'gwei')
+    const permissions = await this.permissions.take(1).toPromise()
+    const app = await this.apps.map(
```

Comment:
It doesn't seem that `app` is needed in the function so it can safely be removed